extensions/extensions-docker/aissemble-jenkins/aissemble-jenkins-agent/src/main/resources/docker/Dockerfile

```diff
@@ -1,3 +1,21 @@
-FROM jenkins/ssh-agent:latest-jdk11
+FROM jenkins/ssh-agent:latest-debian-jdk11
 
-LABEL org.opencontainers.image.source="https://github.com/boozallen/aissemble"
\ No newline at end of file
+LABEL org.opencontainers.image.source="https://github.com/boozallen/aissemble"
+
```

Comment:
I: moved the python configuration to the agent container, where it belongs.  This also improved the speed in which it was installing overall a bit.